p2p/src/test/java/bisq/network/p2p/DummySeedNode.java

```diff
@@ -137,10 +135,11 @@ public void processArgs(String[] args) {
                     checkArgument(arg.contains(":") && arg.split(":").length > 1 && arg.split(":")[1].length() > 3,
                             "Wrong program argument " + arg);
                     List<String> list = Arrays.asList(arg.split(","));
+                    Set<NodeAddress> bannedPeers = new HashSet<>();
```

Comment:
This set isn't used so why even gather the banned peers?

Reply:
Agree. Did just quick adjustment to changes...